apps/gitea/src/main/fabric8/deployment.yml

```diff
@@ -5,7 +5,7 @@ spec:
   template:
     spec:
       containers:
-      - image: fabric8/gitea::${gitea.version}
+      - image: fabric8/gitea:${gitea.version}
```

Comment:
This image doesn't exist under the fabric8 org in docker hub. Is that intentional ?